internal/api/front/perses/src/app/project/prometheusrule/prometheusrule-layout/prometheusrule-layout.component.html

```diff
@@ -1,16 +0,0 @@
-<div [ngClass]="viewDetails ? 'split-container-with-details': 'split-container-without-details'">
-  <as-split direction="horizontal">
-    <as-split-area [size]="gutterStartLocation"
-                   [minSize]="gutterMinLocation"
-                   [maxSize]="gutterMaxLocation">
-      <app-prometheusrule-list></app-prometheusrule-list>
-    </as-split-area>
-    <as-split-area *ngIf="viewDetails"
-                   [size]="100 - gutterStartLocation"
-                   [minSize]="100 - gutterMaxLocation"
-                   [maxSize]="100 - gutterMinLocation">
-      <app-prometheusrule-details></app-prometheusrule-details>
-    </as-split-area>
```

Comment:
I guess we can also remove the dependency in` package.json` regarding this component (`as-split-area`)